src/hipscat/io/file_io/file_pointer.py

```diff
@@ -25,28 +112,32 @@ def append_paths_to_pointer(pointer: FilePointer, *paths: str) -> FilePointer:
     return FilePointer(os.path.join(pointer, *paths))
```

Comment:
Ooo interesting point. like if we were on a windows machine, we'd not want it to do something like `"abfs:///hipscat/data\\windows\\pathing\\to\\hipscat"`... I guess we'd have to test every time we were on a local filesystem for local file construction, and if we weren't on a local filesystem, then construct filepathways in the POSIX manner?

I think since our dependencies don't support Windows (e.g. healpy) ... we shouldn't worry mixing POSIX and nonPOSIX pathways right now.